lib/each.js

```diff
@@ -25,6 +25,7 @@ import wrapAsync from './internal/wrapAsync'
  * If you need the index, use `eachOf`.
  * @param {Function} [callback] - A callback which is called when all
  * `iteratee` functions have finished, or an error occurs. Invoked with (err).
+ * @returns {Promise} a promise, if the callback is omitted
```

Comment:
Deliberately never returning anything from an Async function has given us the headroom to do this!